src/WinGetUtilInterop.UnitTests/TestCollateral/AllEquality.yaml

```diff
@@ -21,7 +21,7 @@ Installers:
       Language: /en-US
       Custom: /s
   - Arch: x64
-    Url: https://contosa.net/publiccontainer/contosainstaller64.exe
+    Url: https://contoso.com/publiccontainer/contosoinstaller64.exe

```

Comment:
I think I'd been ignoring this for a while... contoso.com is microsoft's domain. contosa is a typo of contoso, and contosa.net isn't registered.

If this needs to _not_ be contoso.com, it should be an example.com or some other microsoft controlled domain...